js/audio/Index.svelte

```diff
@@ -104,7 +104,8 @@
 		dragToSeek: true,
 		normalize: true,
 		minPxPerSec: 20,
-		mediaControls: waveform_options.show_controls
+		mediaControls: waveform_options.show_controls,
+		sampleRate: 44100
```

Comment:
I agree with the idea too.
I was going to use this trimming feature for annotation work on a TTS dataset, but I made this change because the voice in the dataset must be 44100Hz by specification.
Realistically, given the use case, I think it would be fine if the sample rate could be specified in waveform_options.

However, I am not familiar with Gradio's internals and am not confident that I can make the changes correctly. 
I would like to leave any further changes to you.